test/recipes/30-test_evp.t

```diff
@@ -33,11 +33,13 @@ push @configs, 'fips.cnf' unless $no_fips;
 
 my @files = qw( evpciph.txt evpdigest.txt );
 my @defltfiles = qw( evpencod.txt evpkdf.txt evppkey_kdf.txt evpmac.txt
-    evppbe.txt evppkey.txt evppkey_ecc.txt evpcase.txt evpaessiv.txt
-    evpccmcavs.txt );
+    evppbe.txt evppkey.txt evppkey_ecc.txt evpcase.txt evpccmcavs.txt );
 my @ideafiles = qw( evpciph_idea.txt );
 push @defltfiles, @ideafiles unless disabled("idea");
 
+my @sivfiles = qw( evpaessiv.txt );
```

Comment:
If you have this file already, might as well make sure it's here

Reply:
The file that already exists..